### ISSUE TITLE
EIG environmental stats

### ISSUE THREAD

**Issue by Impos913** [GTNewHorizons/GT-New-Horizons-Modpack#12064]
### Your GTNH Discord Username

ImpossibleVT#3997

### Your Pack Version

2.2.7

### Your Proposal

The Extreme Industrial Greenhouse should simulate the maximum possible environmental stats for IC2 crops. Because it is supposed to be a greenhouse.
Current environmental stats it is using are;
![image](https://user-images.githubusercontent.com/36479271/209011171-0b0a5bb8-6ce6-4d4d-a131-75b4633b3170.png)

Maximum possible stats being;
![image](https://user-images.githubusercontent.com/36479271/209011438-746003d3-de96-4d6b-ab95-2fffb1aad51b.png)

### Your Goal

In it's current state, on LuV tier EIG, output is very underwhelming and that is because of EIG using bad environmental stats and that means high stat crops growing slower. Therefore it is much slower than advertised speed.
![image](https://user-images.githubusercontent.com/36479271/209012175-31848408-fd11-4fb2-88b6-159de2e6721a.png)
I suspect this to be around 10x instead of 32x speed of a properly set up crop.

### Your Vision

A greenhouse should be a greenhouse and give good environmental stats. Also a LuV tier multi should be better than a LV tier farm.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this change please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members think it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by YTXilo** [comment]
The EIG is balanced for the current tier it is at, you feed it low-tier power, and you get the lower-tier power results, it's not meant to be exactly op if you are feeding it only Luv power, this is also why IC2 crops start at Luv. That's why if you feed it say UV power, you get better results at the sacrifice of more power, at least In my opinion it does not need to be changed. Although the greenhouse thing is true, it does provide better stats than say a bad biome.

**Comment by boubou19** [comment]
buffing the environment stats would incredibly buff it at UEV+.

**Comment by YTXilo** [comment]
Yeah which is my point as buffing it for earlier tier buffs, would significantly affect it for later game purposes. Once it gets to uhv/uev tier it starts to get really good, as also my point, the more power you provide it, the better it gets.

**Comment by Impos913** [comment]
It doesn't even provide any stats. Humidity is from watering the crop, air quality is from seeing the sky and nutrients from applying fertilizer. So it simulates crops as them being in 0 0 0 stat biome.

Also it doesn't even work at the advertised speed because of low environmental stats not working well for high stat crops. I personally see this as a bug.

**Comment by boubou19** [comment]
@kuba6000 what do you think?

**Comment by kuba6000** [comment]
> @kuba6000 what do you think?

I dont think that buff is needed. I'm making Tea fast enough.
